OracleAccessManagement/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Author: Kaushik C
+#
+# Description: script to build a Docker image for Oracle Access Manager 
+#
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+#
+usage() {
+cat << EOF
+
+Usage: buildDockerImage.sh -v [version] -d [domain-type] [-s]
+Builds a Docker Image for Oracle OAM Suite .
+
+Parameters:
+   -h: view usage
+   -v: Release version to build. Required. E.g 12.2.1.4.0
+   -d: Domain type to be built. Required for building the domain oam
```

Comment:
This doesn't make sense to me. What is required?

Reply:
have added -oam , which is required.